test/rpc/test_rpc_transactions.py

```diff
@@ -7,6 +7,10 @@
 from typing import List
 
 from test.account import declare, _get_execute_args, get_nonce, _get_signature
+from test.rpc.conftest import (
+    _prepare_deploy_account_tx,
```

Comment:
Isn't the convention for single-underscore-prepended functions to only be used internally (like within a class or a file)? If so, could you rename these?